src/components/UIComponent/ApparatusInfo.tsx

```diff
@@ -1,17 +1,13 @@
-import { useState } from "react";
-import { ActionData, ExperienceData, SerializedApparatus } from "../../util/types";
+import { useContext, useState } from "react";
+import { SerializedApparatus } from "../../util/types";
 import ApparatusListBox from "../editorBoxes/ApparatusListBox";
 import ActionBox from "../editorBoxes/ActionBox";
+import { GlobalContext, globalContextTypes } from "../../../pages/experience";
 
-type OverlayProps = {
-  experienceData: ExperienceData;
-  addActionToList: (actionData: ActionData) => void;
-  removeActionFromList : (index: number) => void;
+function ApparatusInfo(): JSX.Element {
+  const { experienceData, addActionToList }: globalContextTypes =
```

Comment:
addactiontolist should be used directly by actionbox. (see next comment)

Reply:
same thing for exxperienceData , it is used as a props in one of the components. that props can be removed and context can be used directly in that component

Reply:
> same thing for exxperienceData , it is used as a props in one of the components. that props can be removed and context can be used directly in that component

Not sure which component you are referring to with experience data, all children component of apparatusinfo do not have experience data as a prop